plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -95,11 +95,18 @@ unsafeRunCekNoEmit params mode =
 -- | Evaluate a term using the CEK machine with logging enabled.
 evaluateCek
     :: ( uni `Everywhere` ExMemoryUsage, Ix fun, PrettyUni uni fun)
-    => MachineParameters CekMachineCosts CekValue uni fun
-    -> Term Name uni fun ()
-    -> (Either (CekEvaluationException uni fun) (Term Name uni fun ()), [String])
-evaluateCek params term =
-    case runCek params restrictingEnormous True term of
+    =>
+    -- | Turn on time tracing? If @True@, log comes with timestamp.
+    Bool ->
```

Comment:
Replace `Bool` with `EmitterOptiion` here as well please.

Reply:
Agree, it's easy enough for consumers to work with. We'll need to export the type too, but that's okay.

Reply:
I think just passing it down is simplest.